main.py

```diff
@@ -173,6 +171,7 @@ def get_suffixed_tag(
     key: str,
     suffix: str,
 ) -> tuple[str, str] | tuple[None, None]:
+    # Looks like two style checkers tend to disagree on argument whitespacing.
```

Comment:
What formatter/style checker are you using?
If possible please use [`black`](https://github.com/psf/black) with `--line-length 120`

Maybe it would be a good idea for me to set up [pre-commit](https://pre-commit.com/).